config/config.go

```diff
@@ -1,13 +1,63 @@
 package config
 
-// DebugMode contains boolean of whether application is being run in development
-var DebugMode bool
+import (
+	"github.com/ONSdigital/go-ns/log"
+	"github.com/kelseyhightower/envconfig"
+)
 
-// SiteDomain is the current domain that the router is hosted on
-var SiteDomain = "ons.gov.uk"
+// Configuration structure which hold information for configuring the import API
+type Config struct {
+	BindAddr                 string    `envconfig:"BIND_ADDR"`
+	Debug                    bool      `envconfig:"DEBUG"`
+	SiteDomain               string    `envconfig:"SITE_DOMAIN"`
+	PatternLibraryAssetsPath string    `envconfig:"PATTERN_LIBRARY_ASSETS_PATH"`
+	SupportedLanguages       [2]string `envconfig:"GENERATE_DOWNLOADS_TOPIC"`
+	EnableCookiesControl     bool      `envconfig:"ENABLE_COOKIES_CONTROL"`
+}
 
-// PatternLibraryAssetsPath is the URL to the CSS and JS assets from the pattern library
-var PatternLibraryAssetsPath = "//cdn.ons.gov.uk/sixteens/f816ac8"
+var cfg *Config
 
-// SupportedLanguages is a list of languages that are supported
-var SupportedLanguages = [2]string{"en", "cy"}
+// Get the application and returns the configuration structure
+func Get() (*Config, error) {
+	cfg, err := get()
+	if err != nil {
+		log.Error(err, nil)
```

Comment:
I don't think you need to log here - you can return the error and let the caller decide what to do with it.